x-pack/plugins/security_solution/public/management/components/management_empty_state.tsx

```diff
@@ -0,0 +1,273 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useMemo, MouseEvent, CSSProperties } from 'react';
+import {
+  EuiText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiButton,
+  EuiSteps,
+  EuiTitle,
+  EuiSelectable,
+  EuiProgress,
+  EuiSelectableMessage,
+  EuiSelectableProps,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+const TEXT_ALIGN_CENTER: CSSProperties = Object.freeze({
+  textAlign: 'center',
+});
+
+interface ManagementStep {
+  title: string;
+  children: JSX.Element;
+}
+
+const PolicyEmptyState = React.memo<{
+  loading: boolean;
+  onActionClick: (event: MouseEvent<HTMLAnchorElement | HTMLButtonElement>) => void;
+  actionDisabled: boolean;
+}>(({ loading, onActionClick, actionDisabled }) => {
+  const policySteps = useMemo(
+    () => [
+      {
+        title: i18n.translate('xpack.securitySolution.endpoint.policyList.stepOneTitle', {
+          defaultMessage: 'Head over to Ingest Manager.',
```

Comment:
I know we're still nailing down the labels, but remove period at the end? ( 😬  ) (some for labels below)

Reply:
I'll update all of the labels after we finalize the content for the empty states